### ISSUE TITLE
OBS for Mac

### ISSUE THREAD

**Issue by fopinappb** [alievk/avatarify-python#443]
Just to point out that OBS works better than CamTwist for Mac, now that it is supported in Zoom.
And it's opensource.

**Comment by shehrum** [comment]
Hi @fopinappb , I am having some issues running this with camtwist. The python (avatarify) does not appear for me in camtwist as mentioned in the [steps](https://github.com/alievk/avatarify-python/tree/master/docs). I'm wondering if you had similar issues and if OBS fixed that.

**Comment by fopina** [comment]
yeah I was unable to use CamTwist with Zoom (due to CamTwist not being signed).
OBS worked (it's not signed either but I think it's whitelisted by Zoom)

**Comment by shehrum** [comment]
Hi @fopina , thanks for getting back to me. 

Did you have an issue with the command .run_mac.sh after setting things up on colab or gpu server? 
OR was it straightforward when you followed the instructions. I am stuck at a point where my cam window and avatar window does not appear after i run the command, nor the python (avatarify) appears in the camtwist or OBS:

`./run_mac.sh --is-client --in-addr tcp://6.tcp.ngrok.io:10747 --out-addr tcp://0.tcp.ngrok.io:19906`

Would appreciate any help as I'm stuck in this.

**Comment by fopina** [comment]
I remember having only issues with CamTwist and going with OBS, none with the rest of the instructions, sorry

**Comment by Vazkido** [comment]
> Hi @fopina , thanks for getting back to me. 
> 
> Did you have an issue with the command .run_mac.sh after setting things up on colab or gpu server? 
> OR was it straightforward when you followed the instructions. I am stuck at a point where my cam window and avatar window does not appear after i run the command, nor the python (avatarify) appears in the camtwist or OBS:
> 
> `./run_mac.sh --is-client --in-addr tcp://6.tcp.ngrok.io:10747 --out-addr tcp://0.tcp.ngrok.io:19906`
> 
> Would appreciate any help as I'm stuck in this.
> 


Hey, please help me, if you ever got passed this stage, I'm stuck in this same place  after I run the command, it never open 
camera.

**Comment by SanPatrick01** [comment]
On the 3rd line or so
Change pyaml from 5.1 
 To 6.0.1
